---
title: Philipp Benner <a target="_blank"
			 href="https://pbenner.github.io/">[Website]</a> <a target="_blank" href="https://twitter.com/PhilippBenner2">[Twitter]</a>
description: Researcher
---

<div class="team_members">
	<section>
		<div class="profile_picture">
			<img src=https://bifold.berlin/wp-content/uploads/2020/06/Philipp_Benner_1-450x450.jpg alt="Profile Picture Philipp" class="center">
		</div>
		<div class="personal_description">
			Philipp Benner is a permanent researcher in statistics and machine
			learning interested in materials property prediction. Previously he was a
			postdoctoral researcher in the Computational Molecular Biology Group
			of Prof. Martin Vingron at the Max Planck Institute of Molecular
			Genetics. He worked on statistical and machine learning methods for
			studying gene regulation within the BIFOLD/BZML project
			(bifold.berlin). He did his PhD at the Max Planck Institute for
			Mathematics in the Sciences under the supervision of Prof. Jürgen
			Jost.
		</div>
	</section>
	
	<section>
		<ul class="publications">
			<p>Publications:</p>
			<li>P. Benner, M. Vingron. Quantifying the tissue-specific regulatory information within enhancer DNA sequences. NAR genomics and bioinformatics 3.4 (2021): lqab095.</li>
			<li>P. Benner. Computing Leapfrog Regularization Paths with Applications to Large-Scale K-mer Logistic Regression. Journal of Computational Biology (2021).</li>
			<li>P. Benner, and M. Vingron. ModHMM: A Modular Supra-Bayesian Genome Segmentation Method. Journal of Computational Biology 27.4 (2020): 442-457.</li>
			<li>T. Zehnder, P. Benner, and M. Vingron. Predicting enhancers in mammalian genomes using supervised hidden Markov models. BMC Bioinformatics 20(1):157, 2019</li>
			<li>A. Ramisch, V. Heinrich, ..., P. Benner, ... CRUP: A comprehensive framework to predict condition-specific regulatory units. Genome Biol 20, 227 (2019) doi:10.1186/s13059-019-1860-7</li>
			<li>S. Schöne, M. Bothe, E. Einfeldt, M. Borschiwer and P. Benner, M. Vingron, Martin, M. Thomas-Chollier, and S. Meijsing. Synthetic STARR-seq reveals how DNA shape and sequence modulate transcriptional output and noise. PLoS Genetics, 14(11):e1007793, 2018</li>
			<li>P. Benner. “Combining Prior Information for the Prediction of Transcription Factor Binding Sites”. PhD thesis. Universität Leipzig, 2016. url: https://nbn-resolving.org/urn:nbn:de:bsz:15-qucosa2-215418.</li>
			<li>P. Benner, M. Bacák, and P.-Y. Bourguignon. Point estimates in phylogenetic reconstructions. Bioinformatics, 30(17):i534-i540, 2014.</li>
			<li>P. Benner and S. Poppe. Stochastische Prozesse und Bayessches Schätzen. In Nicole J. Saam and Norman Braun, editors, Handbuch Modellbildung und Simulation in den Sozialwissenschaften, Springer, 2014.</li>
			<li>S. Poppe, P. Benner, and T. Elze. A predictive approach to nonparametric inference for adaptive sequential sampling of psychophysical experiments. Journal of Mathematical Psychology, 56(3):179–195, 2012.</li>
			<li>R. Stoop, S. Martignoli, P. Benner, R. Stoop, and Y. Uwate. Shrimps: Occurrence, scaling and relevance. International Journal of Bifurcation and Chaos, 22(10), 2012.</li>
			<li>R. Stoop, P. Benner, and Y. Uwate. Real-world existence and origins of global shrimp organization on spirals. Physical Review Letters, 105(7):074102, 2010.</li>
		</ul>
	</section>
</div>
