---
title: Thilo Muth
description: Former head of eScience group
---

<div class="team_members">
	<section>
		<div class="profile_picture">
			<img src="/images/team_pictures/thilo.jpg" alt="Profile Picture Thilo" class="center">
		</div>
		<div class="personal_description">
			Thilo Muth leads the eScience group. He studied bioinformatics at Friedrich Schiller University Jena, followed by a PhD in the bioprocess engineering group of Prof. Udo Reichl at the Max Planck Institute Magdeburg. In parallel to his PhD he worked as the lead software engineer for the glycoanalytical company GlyXera GmbH. In his PhD thesis, he evaluated and developed bioinformatics methods for protein analytics of microbial communities. After his PhD he worked as a team leader in the Bioinformatics Unit at the Robert Koch Institute in Berlin. Since 2017 he regularly teaches in the Bioinformatics and lately Data Science Master’s courses at Free University (FU) Berlin.
			In general, he is interested in all sorts of digitalization tasks that lead to changes in an organization. From his perspective, the focus of the eScience group is to find and solve challenging problems in interdisciplinary work with colleagues from different scientific domains using data management and data science techniques. He also really likes coding in Java and Python and once in a while works on some pet projects…	
		</div>
	</section>
	<!-- <h3>Social media:</h3> -->

		<div class="socials">
			<a target="_blank" href="https://orcid.org/0000-0001-8304-2684">ORCID</a>
			<br>
			<a target="_blank" href="https://www.linkedin.com/in/thilo-muth-18353319a/">LinkedIn</a>
			<br>
			<a target="_blank" href="https://twitter.com/thilo_muth">Twitter</a>
		</div>
	
	<section>
		<ul class="publications">
			<p>Publications:</p>
			<li>Oveland E, Muth T, Rapp E, Martens L, Berven FS, Barsnes H. Viewing the proteome: how to visualize proteomics data? Proteomics. 2015 Apr;15(8):1341-55. doi: 10.1002/pmic.201400412. Epub 2015 Jan 21. PMID: 25504833.</li>
			<li>Seifert J, Muth T. Editorial for Special Issue: Metaproteomics. Proteomes. 2019 Mar 5;7(1):9. doi: 10.3390/proteomes7010009. PMID: 30841491; PMCID: PMC6473379.</li>
			<li>Brown P; RELISH Consortium, Zhou Y. Large expert-curated database for benchmarking document similarity detection in biomedical literature search. Database (Oxford). 2019 Jan 1;2019:baz085. doi: 10.1093/database/baz085. PMID: 33326193; PMCID: PMC7291946.</li>
			<li>Grossegesse M, Hartkopf F, Nitsche A, Schaade L, Doellinger J, Muth T. Perspective on Proteomics for Virus Detection in Clinical Samples. J Proteome Res. 2020 Nov 6;19(11):4380-4388. doi: 10.1021/acs.jproteome.0c00674. Epub 2020 Oct 22. PMID: 33090795; PMCID: PMC7640980.</li>
			<li>Sajulga R, Easterly C, Riffle M, Mesuere B, Muth T, Mehta S, Kumar P, Johnson J, Gruening BA, Schiebenhoefer H, Kolmeder CA, Fuchs S, Nunn BL, Rudney J, Griffin TJ, Jagtap PD. Survey of metaproteomics software tools for functional microbiome analysis. PLoS One. 2020 Nov 10;15(11):e0241503. doi: 10.1371/journal.pone.0241503. PMID: 33170893; PMCID: PMC7654790.</li>
			<li>Schiebenhoefer H, Van Den Bossche T, Fuchs S, Renard BY, Muth T, Martens L. Challenges and promise at the interface of metaproteomics and genomics: an overview of recent progress in metaproteogenomic data analysis. Expert Rev Proteomics. 2019 May;16(5):375-390. doi: 10.1080/14789450.2019.1609944. Epub 2019 Apr 30. PMID: 3100254</li>
			<li>Van Den Bossche T, Verschaffelt P, Schallert K, Barsnes H, Dawyndt P, Benndorf D, Renard BY, Mesuere B, Martens L, Muth T. Connecting MetaProteomeAnalyzer and PeptideShaker to Unipept for Seamless End-to-End Metaproteomics Data Analysis. J Proteome Res. 2020 Aug 7;19(8):3562-3566. doi: 10.1021/acs.jproteome.0c00136. Epub 2020 Jun 11. PMID: 3243114</li>
			<li>Fischer M, Muth T, Renard BY. Peptide-to-Protein Summarization: An Important Step for Accurate Quantification in Label-Based Proteomics. Methods Mol Biol. 2019;1977:159-180. doi: 10.1007/978-1-4939-9232-4_11. PMID: 3098032</li>
			<li>Schiebenhoefer H, Schallert K, Renard BY, Trappe K, Schmid E, Benndorf D, Riedel K, Muth T, Fuchs S. A complete and flexible workflow for metaproteomics data analysis based on MetaProteomeAnalyzer and Prophane. Nat Protoc. 2020 Oct;15(10):3212-3239. doi: 10.1038/s41596-020-0368-7. Epub 2020 Aug 28. PMID: 328599</li>
			<li>Kuhring M, Doellinger J, Nitsche A, Muth T, Renard BY. TaxIt: An Iterative Computational Pipeline for Untargeted Strain-Level Identification Using MS/MS Spectra from Pathogenic Single-Organism Samples. J Proteome Res. 2020 Jun 5;19(6):2501-2510. doi: 10.1021/acs.jproteome.9b00714. Epub 2020 May 15. PMID: 323621</li>
			<li>Lechner J, Hartkopf F, Hiort P, Nitsche A, Grossegesse M, Doellinger J, Renard BY, Muth T. Purple: A Computational Workflow for Strategic Selection of Peptides for Viral Diagnostics Using MS-Based Targeted Proteomics. Viruses. 2019 Jun 8;11(6):536. doi: 10.3390/v11060536. PMID: 31181768; PMCID: PMC66309</li>
			<li>Heyer R, Schallert K, Büdel A, Zoun R, Dorl S, Behne A, Kohrs F, Püttker S, Siewert C, Muth T, Saake G, Reichl U, Benndorf D. A Robust and Universal Metaproteomics Workflow for Research Studies and Routine Diagnostics Within 24 h Using Phenol Extraction, FASP Digest, and the MetaProteomeAnalyzer. Front Microbiol. 2019 Aug 16;10:1883. doi: 10.3389/fmicb.2019.01883. PMID: 31474963; PMCID: PMC6707425.</li>
			<li>Muth T, Hartkopf F, Vaudel M, Renard BY. A Potential Golden Age to Come- Current Tools, Recent Use Cases, and Future Avenues for De Novo Sequencing in Proteomics. Proteomics. 2018 Sep;18(18):e1700150. doi: 10.1002/pmic.201700150. Epub 2018 Aug 22. PMID: 29968278.</li>
			<li>Muth T, Rapp E, Berven FS, Barsnes H, Vaudel M. Tandem Mass Spectrum Sequencing: An Alternative to Database Search Engines in Shotgun Proteomics. Adv Exp Med Biol. 2016;919:217-226. doi: 10.1007/978-3-319-41448-5_10. PMID: 27975219.</li>
			<li>Blank C, Easterly C, Gruening B, Johnson J, Kolmeder CA, Kumar P, May D, Mehta S, Mesuere B, Brown Z, Elias JE, Hervey WJ, McGowan T, Muth T, Nunn B, Rudney J, Tanca A, Griffin TJ, Jagtap PD. Disseminating Metaproteomic Informatics Capabilities and Knowledge Using the Galaxy-P Framework. Proteomes. 2018 Jan 31;6(1):7. doi: 10.3390/proteomes6010007. PMID: 29385081; PMCID: PMC5874766.</li>
			<li>Muth T, Renard BY, Martens L. Metaproteomic data analysis at a glance: advances in computational microbial community proteomics. Expert Rev Proteomics. 2016 Aug;13(8):757-69. doi: 10.1080/14789450.2016.1209418. Epub 2016 Jul 20. PMID: 27376173.</li>
			<li>Muth T, Benndorf D, Reichl U, Rapp E, Martens L. Searching for a needle in a stack of needles: challenges in metaproteomics data analysis. Mol Biosyst. 2013 Apr 5;9(4):578-85. doi: 10.1039/c2mb25415h. PMID: 23238088.</li>
			<li>Muth T, Renard BY. Evaluating de novo sequencing in proteomics: already an accurate alternative to database-driven peptide identification? Brief Bioinform. 2018 Sep 28;19(5):954-970. doi: 10.1093/bib/bbx033. PMID: 28369237.</li>
			<li>Muñoz-Benavent M, Hartkopf F, Van Den Bossche T, Piro VC, García-Ferris C, Latorre A, Renard BY, Muth T. gNOMO: a multi-omics pipeline for integrated host and microbiome analysis of non-model organisms. NAR Genom Bioinform. 2020 Aug 5;2(3):lqaa058. doi: 10.1093/nargab/lqaa058. Erratum in: NAR Genom Bioinform. 2020 Oct 09;2(4):lqaa083. PMID: 33575609; PMCID: PMC7671378.</li>
			<li>Muth T, Kolmeder CA, Salojärvi J, Keskitalo S, Varjosalo M, Verdam FJ, Rensen SS, Reichl U, de Vos WM, Rapp E, Martens L. Navigating through metaproteomics data: a logbook of database searching. Proteomics. 2015 Oct;15(20):3439-53. doi: 10.1002/pmic.201400560. Epub 2015 Apr 27. PMID: 25778831.</li>
			<li>Muth T, Kohrs F, Heyer R, Benndorf D, Rapp E, Reichl U, Martens L, Renard BY. MPA Portable: A Stand-Alone Software Package for Analyzing Metaproteome Samples on the Go. Anal Chem. 2018 Jan 2;90(1):685-689. doi: 10.1021/acs.analchem.7b03544. Epub 2017 Dec 19. PMID: 29215871; PMCID: PMC5757220.</li>
			<li>Anzt H, Bach F, Druskat S, Löffler F, Loewe A, Renard BY, Seemann G, Struck A, Achhammer E, Aggarwal P, Appel F, Bader M, Brusch L, Busse C, Chourdakis G, Dabrowski PW, Ebert P, Flemisch B, Friedl S, Fritzsch B, Funk MD, Gast V, Goth F, Grad JN, Hegewald J, Hermann S, Hohmann F, Janosch S, Kutra D, Linxweiler J, Muth T, Peters-Kottig W, Rack F, Raters FHC, Rave S, Reina G, Reißig M, Ropinski T, Schaarschmidt J, Seibold H, Thiele JP, Uekermann B, Unger S, Weeber R. An environment for sustainable research software in Germany and beyond: current state, open challenges, and call for action. F1000Res. 2020 Apr 27;9:295. doi: 10.12688/f1000research.23224.2. PMID: 33552475; PMCID: PMC7845155.</li>
			<li>Muth T, Behne A, Heyer R, Kohrs F, Benndorf D, Hoffmann M, Lehtevä M, Reichl U, Martens L, Rapp E. The MetaProteomeAnalyzer: a powerful open-source software suite for metaproteomics data analysis and interpretation. J Proteome Res. 2015 Mar 6;14(3):1557-65. doi: 10.1021/pr501246w. Epub 2015 Feb 23. PMID: 25660940.</li>
			<li>Kolmeder CA, Ritari J, Verdam FJ, Muth T, Keskitalo S, Varjosalo M, Fuentes S, Greve JW, Buurman WA, Reichl U, Rapp E, Martens L, Palva A, Salonen A, Rensen SS, de Vos WM. Colonic metaproteomic signatures of active bacteria and the host in obesity. Proteomics. 2015 Oct;15(20):3544-52. doi: 10.1002/pmic.201500049. Epub 2015 Sep 11. PMID: 26255997.</li>
			<li>Tanca A, Palomba A, Fraumene C, Pagnozzi D, Manghina V, Deligios M, Muth T, Rapp E, Martens L, Addis MF, Uzzau S. The impact of sequence database choice on metaproteomic results in gut microbiota studies. Microbiome. 2016 Sep 27;4(1):51. doi: 10.1186/s40168-016-0196-8. PMID: 27671352; PMCID: PMC5037606.</li>
			<li>Muth T, Weilnböck L, Rapp E, Huber CG, Martens L, Vaudel M, Barsnes H. DeNovoGUI: an open source graphical user interface for de novo sequencing of tandem mass spectra. J Proteome Res. 2014 Feb 7;13(2):1143-6. doi: 10.1021/pr4008078. Epub 2014 Jan 7. PMID: 24295440; PMCID: PMC3923451.</li>
			<li>Muñoz-Benavent M, Hartkopf F, Van Den Bossche T, Piro VC, García-Ferris C, Latorre A, Renard BY, Muth T. Erratum: gNOMO: a multi-omics pipeline for integrated host and microbiome analysis of non-model organisms. NAR Genom Bioinform. 2020 Oct 9;2(4):lqaa083. doi: 10.1093/nargab/lqaa083. Erratum for: NAR Genom Bioinform. 2020 Aug 05;2(3):lqaa058. PMID: 33577626; PMCID: PMC7671335.</li>
			<li>Huffman JE, Pučić-Baković M, Klarić L, Hennig R, Selman MH, Vučković F, Novokmet M, Krištić J, Borowiak M, Muth T, Polašek O, Razdorov G, Gornik O, Plomp R, Theodoratou E, Wright AF, Rudan I, Hayward C, Campbell H, Deelder AM, Reichl U, Aulchenko YS, Rapp E, Wuhrer M, Lauc G. Comparative performance of four methods for high-throughput glycosylation analysis of immunoglobulin G in genetic and epidemiological research. Mol Cell Proteomics. 2014 Jun;13(6):1598-610. doi: 10.1074/mcp.M113.037465. Epub 2014 Apr 9. PMID: 24719452; PMCID: PMC4047478.</li>
			<li>Muth T, Vaudel M, Barsnes H, Martens L, Sickmann A. XTandem Parser: an open- source library to parse and analyse X!Tandem MS/MS search results. Proteomics. 2010 Apr;10(7):1522-4. doi: 10.1002/pmic.200900759. Erratum in: Proteomics. 2010 Jul;10(13):2560. PMID: 20140905.</li>
			<li>Muth T, García-Martín JA, Rausell A, Juan D, Valencia A, Pazos F. JDet: interactive calculation and visualization of function-related conservation patterns in multiple sequence alignments and structures. Bioinformatics. 2012 Feb 15;28(4):584-6. doi: 10.1093/bioinformatics/btr688. Epub 2011 Dec 9. PMID: 22171333.</li>
			<li>Behne A, Muth T, Borowiak M, Reichl U, Rapp E. glyXalign: high-throughput migration time alignment preprocessing of electrophoretic data retrieved via multiplexed capillary gel electrophoresis with laser-induced fluorescence detection-based glycoprofiling. Electrophoresis. 2013 Aug;34(16):2311-5. doi: 10.1002/elps.201200696. Epub 2013 Jul 8. PMID: 23637070.</li>
			<li>Muth T, Peters J, Blackburn J, Rapp E, Martens L. ProteoCloud: a full- featured open source proteomics cloud computing pipeline. J Proteomics. 2013 Aug 2;88:104-8. doi: 10.1016/j.jprot.2012.12.026. Epub 2013 Jan 8. PMID: 23305951.</li>
			<li>Pech S, Rehberg M, Janke R, Benndorf D, Genzel Y, Muth T, Sickmann A, Rapp E, Reichl U. Tracking changes in adaptation to suspension growth for MDCK cells: cell growth correlates with levels of metabolites, enzymes and proteins. Appl Microbiol Biotechnol. 2021 Mar;105(5):1861-1874. doi: 10.1007/s00253-021-11150-z. Epub 2021 Feb 13. PMID: 33582836; PMCID: PMC7907048.</li>
			<li>Helsens K, Colaert N, Barsnes H, Muth T, Flikka K, Staes A, Timmerman E, Wortelkamp S, Sickmann A, Vandekerckhove J, Gevaert K, Martens L. ms_lims, a simple yet powerful open source laboratory information management system for MS- driven proteomics. Proteomics. 2010 Mar;10(6):1261-4. doi: 10.1002/pmic.200900409. Erratum in: Proteomics.2010 Jul;10(13):2560. PMID: 20058248.</li>
			<li>Muth T, Keller D, Puetz SM, Martens L, Sickmann A, Boehm AM. jTraqX: a free, platform independent tool for isobaric tag quantitation at the protein level. Proteomics. 2010 Mar;10(6):1223-5. doi: 10.1002/pmic.200900374. PMID: 20058250.	</li>
		</ul>
	</section>
</div>
